src/main/java/com/minecolonies/coremod/colony/interactionhandling/RequestBasedInteraction.java

```diff
@@ -150,7 +150,10 @@ public void onWindowOpened(final Window window, final ICitizenDataView dataView)
 
                 icon.setID("request_item");
                 icon.setSize(32, 32);
-                icon.setItem((request.getDisplayStacks().get(0)));
+                if (!request.getDisplayStacks().isEmpty())
+                {
+                    icon.setItem((request.getDisplayStacks().get(0)));
```

Comment:
might want to include the `   icon.setPosition(30, 60);
                icon.setVisible(true);` aswell, to not display Air?